content/docs/user-guide/concepts/dvc-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+name: 'DVC Cache'
+match: ['DVC cache', 'cache', 'caches', 'cached']
+tooltip: >-
+  The [DVC cache](/doc/user-guide/concepts/dvc-cache) is a hidden storage (by
+  default located in the `.dvc/cache` directory) for files that are tracked by
+  DVC, and their different versions.
+description: >-
+  The DVC cache adds a layer of indirection between code and data to efficiently
+  version large datasets, data science features, and machine learning models.
```

Comment:
Same general Qs as with data-pipelines: what terms did the SEO/search history research throw for this one?

So far all the descriptions seem like excerpts from previous content we had. I was looking for some new stuff based on SEO research.

Can we list the terms/goals for each concept's SEO desc? Thanks

Reply:
> Looks like SEO metadata is missing from the dependency, dvc-cache, dvc-metafiles, output concepts (there are no changes to those files). All the frontmatter in fact (inc. draft tooltip)

Not following you here... dependency and output are glossary terms but aren't in the list of concepts. For dvc-cache, it is a full concept but it does have the frontmatter in the latest. 🤔  So I was only going to add frontmatter to dvc-metafiles. Is that different from what you were thinking @jorgeorpinel ?

Reply:
Ping @jeremydesroches

Reply:
> I was looking for some new stuff based on SEO research.

> Can we list the terms/goals for each concept's SEO desc?

I added the last half of the description based on the terms, which are listed in the HTML comments.

Reply:
I see. Alright.